src/Symfony/Bundle/FrameworkBundle/Resources/config/console.xml

```diff
@@ -12,5 +12,63 @@
             <tag name="kernel.event_subscriber" />
             <tag name="monolog.logger" channel="console" />
         </service>
+
+        <service id="command.about" class="Symfony\Bundle\FrameworkBundle\Command\AboutCommand">
```

Comment:
For the record and what it's worth, the pass use `console.command` tag & prefix.

`web_server.command.server_run` looks ok to me, as we already have a root prefix. So there is few chances to be used in userland.
`security.console.user_password_encoder_command` was not conform to this last convention, but the web server bundle didn't exist yet nor any convention for command as service at the time it was introduced. So I used mine, i.e: `(namespace|context).component.snaked_case_class_name` :)

Finally, `command` as root prefix might mean something else in userland. It can be one of your domain bounded context you use as prefix for related services, or simply the prefix you chose for your command handlers when using command buses.

Actually, I don't really mind either; there are few chances there is any conflict with any existing app. Just saying it won't hurt having this prefix :)